docs/sdk/client-side-sdks/android/android-usage.md

```diff
@@ -87,6 +87,15 @@ Map<String, BaseConfigVariable> variables = devcycleClient.allVariables();
 
 If the SDK has not finished initializing, these methods will return an empty Map.
 
+:::info
```

Comment:
I wonder if these should be a warning? If someone starts using this hook in productions - bad things could happen...

Reply:
What I mean here is a "Danger" callout vs. "Info" callout.